tests/suites/0_stateless/13_tpch/13_0022_q22.sql

```diff
@@ -47,4 +47,4 @@ drop table region;
 drop table supplier;
 drop table part;
 drop table partsupp;
-set enable_planner_v2 = 0;
\ No newline at end of file
+set enable_planner_v2 = 0;
```

Comment:
Planner v2 is enabled by default, no need set this setting anymore

Reply:
ok